src/simplay-web/src/SimulationData.ts

```diff
@@ -0,0 +1,44 @@
+import { Entity } from './Entity';
+import { Event } from './Event';
+import { EventFactory } from './EventFactory';
+import { SimplayGrid } from './SimplayGrid';
+import { SimulationDataSerialized } from './SimulationDataSerialized';
+import { Sprite } from './Sprite';
+import { Visual } from './Visual';
+
+export class SimulationData {
+  readonly events: Event[];
+  readonly grid: SimplayGrid;
+  readonly entities: Entity[];
+  readonly visuals: Visual[];
+  readonly sprites: Sprite[];
+
+  constructor(
```

Comment:
A user is expected to call `new SimulationSpooler(SimulationData)`: https://github.com/team-simplay/SimPlay/pull/8/files#diff-640577f20a31fa91afd9ce5393ed6bdadead89a249ace356c3f4cf73edf48c48R12

This means the user is also supposed to call `SimulationData.fromSerialized(SimulationDataSerialized);`. Can we change that so that the user only has to make one call to our API, instead of `SimulationData`, the `SimulationSpooler` should use `SimulationDataSerialized` in the constructor?